Non-ASCII text shows as mojibake on lines containing wiki markup (Jira Server)

**Describe the bug**

When viewing a Jira Server / Data Center issue, multibyte UTF-8 text (Greek, Cyrillic, CJK, accented Latin, emoji) renders as mojibake — but only on lines that also contain Jira-wiki markup. For example, the Greek word Ολοκληρώθηκε displays as ÎÎ»Î¿ÎºÎ»Î·ÏÏÎ¸Î·ÎºÎµ.

Lines without any wiki markup render the same text correctly, which makes the breakage look inconsistent within a single issue.

**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="1.7.0", GitCommit="Homebrew", CommitDate="2025-08-30T08:30:25+00:00", GoVersion="go1.25.5", Compiler="gc", Platform="darwin/arm64")
    ```
2. Are you using Jira cloud or on-premise jira server?
    ```
    SERVER INFO
    -----------
    
    Version: 	 10.3.22
    Build Number: 	 10030022
    Deployment Type: Server
    Default Locale:  
    ```
3. What operating system are you using? Also mention version.
    ```
    macOS Tahoe 26.5.1
    ```
4. What terminal are you using? Also mention version.
    ```
    Ghostty 1.3.1 (Build 15212)
    ```

**To Reproduce**

  This reproduces with no Jira server — the bug is in the Jira-wiki → markdown parser (pkg/md/jirawiki.Parse).

  Add a quick test:

```go
  // pkg/md/jirawiki/repro_test.go
  package jirawiki

  import "testing"

  func TestReproMojibake(t *testing.T) {
        got := Parse("Λέξη με *bold* κείμενο.")
        want := "Λέξη με **bold** κείμενο.\n"
        if got != want {
                t.Errorf("\n got: %q\nwant: %q", got, want)
        }
  }
```
  Run it:

```bash
  go test ./pkg/md/jirawiki/ -run TestReproMojibake
```

  Observed output:

```
--- FAIL: TestReproMojibake (0.00s)
    repro_test.go:10:
         got: "Î\u009bÎ\u00adÎ¾Î· Î¼Îµ **bold** ÎºÎµÎ¯Î¼ÎµÎ½Î¿.\n"
        want: "Λέξη με **bold** κείμενο.\n"
FAIL
FAIL    github.com/ankitpokhrel/jira-cli/pkg/md/jirawiki        0.009s
FAIL
```

  The non-ASCII text is corrupted, while the ASCII markup (*bold* → **bold**) converts fine.

**Expected behavior**
Non-ASCII text in descriptions and comments should render correctly whether or not the line also contains Jira-wiki markup.

--- a/pkg/md/jirawiki/parser.go
+++ b/pkg/md/jirawiki/parser.go
@@ -184,7 +184,7 @@ func secondPass(lines []string) string {
 					end = token.endIdx
 				}
 			} else {
-				out.WriteRune(rune(line[beg]))
+				out.WriteByte(line[beg])
 			}
 
 			end++

--- a/pkg/md/jirawiki/parser_test.go
+++ b/pkg/md/jirawiki/parser_test.go
@@ -615,3 +615,49 @@ func TestTables(t *testing.T) {
 		})
 	}
 }
+
+func TestParseNonASCIIText(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
+			name:     "greek with bold",
+			input:    "Λέξη με *bold* κείμενο.",
+			expected: "Λέξη με **bold** κείμενο.\n",
+		},
+		{
+			name:     "cyrillic with bold",
+			input:    "Текст с *bold* словом.",
+			expected: "Текст с **bold** словом.\n",
+		},
+		{
+			name:     "cjk with bold",
+			input:    "これは *bold* です。",
+			expected: "これは **bold** です。\n",
+		},
+		{
+			name:     "accented latin with reference link",
+			input:    "Café [link|https://ankit.pl] señor.",
+			expected: "Café [link](https://ankit.pl) señor.\n",
+		},
+		{
+			name:     "emoji with bold",
+			input:    "Status 🚀 *done* ✅",
+			expected: "Status 🚀 **done** ✅\n",
+		},
+	}
+
+	for _, tc := range cases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+
+			assert.Equal(t, tc.expected, Parse(tc.input))
+		})
+	}
+}

